Classes/Controller/EntityFactController.php

```diff
@@ -77,9 +77,9 @@ public function showAction()
 
         //write the user sorted selection given by Flexform into another helper array (special customer request to select specific entities from sameAs)
         $multiSelectArray2 = explode(",",($this->settings['entityfacts']['sameAsSelected']));
-
+        
```

Comment:
No need to add spaces/tabs here. Please check your IDE for setting "trim trailing whitespace on save".

In .editorconfig this would be
`trim_trailing_whitespace=true`